site/src/pages/CreateWorkspacePage/CreateWorkspacePageView.stories.tsx

```diff
@@ -382,3 +382,43 @@ export const ExternalAuthAllConnected: Story = {
 		],
 	},
 };
+
+export const WithViewSourceButton: Story = {
+	args: {
+		canUpdateTemplate: true,
+		versionId: "template-version-123",
+		template: {
+			...MockTemplate,
+			organization_name: "default",
+			name: "docker-template",
+		},
+	},
+	parameters: {
+		docs: {
+			description: {
+				story:
+					"This story shows the View Source button that appears for template administrators. The button allows quick navigation to the template editor from the workspace creation page.",
+			},
+		},
+	},
+};
+
+export const WithoutViewSourceButton: Story = {
+	args: {
+		canUpdateTemplate: false,
+		versionId: "template-version-123",
+		template: {
+			...MockTemplate,
+			organization_name: "default",
+			name: "docker-template",
+		},
+	},
+	parameters: {
+		docs: {
+			description: {
+				story:
+					"This story shows the workspace creation page for users without template update permissions. The View Source button is hidden for these users.",
+			},
+		},
+	},
+};
```

Comment:
you don't need a story for this. the default is false so every other story already verifies this.